templates/packer/ansible/roles/common/files/worker-initialization-scripts/client_credentials.sh

```diff
@@ -20,6 +20,7 @@ jq -r ".client_secret" /tmp/client-credentials.json > /etc/osbuild-worker/client
 rm -f /tmp/client-credentials.json
 
 sudo tee -a /etc/osbuild-worker/osbuild-worker.toml > /dev/null << EOF
+deployment_channel = "${CHANNEL:-local}"
```

Comment:
@croissanne can you please give me some hints where to add this additionally?
as I think this will only hand it over to one set or workers but not all tenants

Reply:
This will work for the fedora koji workers, service workers, and community workers.